config/exercise_readme.go.tmpl

```diff
@@ -62,6 +62,10 @@ To see, if your solution contains some common ineffective use cases, inside the
 cargo clippy --all-targets
 ```
 
+## Submitting the solution
+
+Generally you want to submit the files in which you have implemented your solution (`src/lib.rs` in most cases). If you are using any external crates, please consider submitting the `Cargo.toml` file in order for the mentor to learn which version of the library you are using. This will make the review process faster and clearer.
```

Comment:
Both reasons for submitting `Cargo.toml` are valid. I therefore propose the King Solomon solution:

```suggestion
Generally you should submit all files in which you implemented your solution (`src/lib.rs` in most cases). If you are using any external crates, please consider submitting the `Cargo.toml` file. This will make the review process faster and clearer.
```